cmd/sonobuoy/app/gen_test.go

```diff
@@ -186,10 +190,13 @@ func TestResolveConfig(t *testing.T) {
 				Aggregation:      plugin.AggregationConfig{TimeoutSeconds: 100},
 				Resources:        config.DefaultResources,
 			},
+			expectedGenConfigPlugins: &client.GenConfig{
+				DynamicPlugins: []string{"e2e", "systemd-logs"},
+			},
 		}, {
-			name:     "Flags shouldn't override the config settings unless set",
-			input:    &genFlags{},
-			cliInput: "--sonobuoy-image=flagImage --config testdata/sonobuoy.conf",
+			name:               "Flags shouldn't override the config settings unless set",
+			input:              "--sonobuoy-image=flagImage --config " + dynamicConfigFileName,
+			configFileContents: `{"Namespace":"configNS","WorkerImage":"configImage","ImagePullPolicy":"Never","Server":{"timeoutseconds":500}}`,
```

Comment:
This `sonobuoy.conf` was used just for a few of these tests and since I already added the logic to make/use a temp one I just deleted the file and copied the contents to a few tests.

Much more flexible and now it is clear what the conf file is for each test instead of having to go look it up.